cibuildwheel/macos.py

```diff
@@ -460,7 +460,7 @@ def build(options: Options, tmp_path: Path) -> None:
                     env=env,
                     capture_stdout=True,
                 ).strip()
-                testing_archs: list[Literal["x86_64", "arm64"]]
+                testing_archs: list[Literal["x86_64", "arm64"]]  # noqa: F821
```

Comment:
(Please feel free to keep filing stuff as questions and issues come up. Glad we could resolve this one :))

Reply:
It's not resolved, it's just a duplicate of an existing issue (https://github.com/charliermarsh/ruff/issues/2005) you helped me open a couple of days ago. :)

Reply:
It will go away (Ruff will actually remove it) when this does get fixed, so fine to leave it in here. But we could probably turn off F821 as well, since we are fully typed. It's probably mostly useful if you have code not checked by MyPy.